shared/chat/conversation/messages/coinflip/container.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import CoinFlip from '.'
+import * as RPCChatTypes from '../../../../constants/types/rpc-chat-gen'
+import {namedConnect} from '../../../../util/container'
+
+type OwnProps = {|flipGameID: string|}
+
+const participantStatuses = [
+  RPCChatTypes.chatUiUICoinFlipPhase.commitment,
+  RPCChatTypes.chatUiUICoinFlipPhase.reveals,
+  RPCChatTypes.chatUiUICoinFlipPhase.complete,
+]
+
+const mapStateToProps = (state, {flipGameID}: OwnProps) => {
+  const status = state.chat2.getIn(['flipStatusMap', flipGameID])
+  return !status
+    ? {
+        isError: false,
+        participants: [],
```

Comment:
Make a const or this render thrashes

Reply:
Would be nice if there was like a global blank list